pkg/kyma/cmd/provision/minikube/common.go

```diff
@@ -35,36 +35,43 @@ func promptUser() bool {
 }
 
 func addDevDomainsToEtcHostsOSSpecific(o *MinikubeOptions, s step.Step, hostAlias string) error {
-	notifyUserFunc := func() {
-		s.LogInfof("Execute the following command manually to add mappings: sudo sed -i '' \"/"+o.Domain+"/d\" "+hostsFile+" && echo '%s' | sudo tee -a /etc/hosts\r\n", hostAlias)
+	notifyUserFunc := func(err error) {
+		if err != nil {
+			s.LogInfof("Error: %s", err.Error())
+		}
+		s.LogInfof("Execute the following command manually to add domain entries: sudo sed -i '' \"/"+o.Domain+"/d\" "+hostsFile+" && echo '%s' | sudo tee -a /etc/hosts\r\n", hostAlias)
 		return
```

Comment:
I know you did not change this line, but this `return` statement is pointless, it can just be deleted.

The function does not return anything and it is also not breaking out of any control structure.

Reply:
Yes, but in general, I think it is a good practice to mark the exit explicitly. Anyway, I removed that for now.